app/config/projects.yml

```diff
@@ -33,6 +33,12 @@ parameters:
                     name: '2.5'
                     branchName: '2.5'
                     slug: '2.5'
+                    maintained: false
```

Comment:
It is for sure possible. We could standardize a file name and format for me to look for in the code repo to pull in project information.

Reply:
this is exactly what I was thinking of. might be good future improvement :+1:

Reply:
I can do that. I will look for a `project.json` file in the root of the project and it can provide data about the project. Initially it won't exist but if you introduce it to the project, it will take precedent over the data provided by doctrine-website and we can slowly migrate to this new concept.